test/functional/page_objects/visual_builder_page.js

```diff
@@ -66,6 +66,36 @@ export function VisualBuilderPageProvider({ getService, getPageObjects }) {
       await metricValue.session.moveMouseTo(metricValue);
       return await metricValue.getVisibleText();
     }
+
+    async clickGauge() {
+      await testSubjects.click('gaugeTsvbTypeBtn');
+    }
+
+    async getGaugeLabel() {
+      const gaugeLabel = await find.byCssSelector('.thorHalfGauge__label');
+      return await gaugeLabel.getVisibleText();
+    }
+
+    async getGaugeCount() {
+      const gaugeCount = await find.byCssSelector('.thorHalfGauge__value');
+      return await gaugeCount.getVisibleText();
+    }
+
+    async clickTopN()
+    {
+      await testSubjects.click('top_nTsvbTypeBtn');
+    }
```

Comment:
Mentioned above, but please add `await PageObjects.header.waitUntilLoadingHasFinished();` for each of these `clickSomeTab` methods so that we should be able to remove the sleeps.